src/components/IPScan/Actions/Group/ReRun/index.tsx

```diff
@@ -45,22 +46,14 @@ const ReRun = ({ jobsData, goToCustomLocation }: Props) => {
   };
 
   return (
-    <Tooltip
-      title={
-        <div>
-          Start a new sequence search using the same sequences as this job.
-        </div>
-      }
+    <Button
+      type={jobsPage ? 'inline' : 'secondary'}
+      onClick={handleReRun}
+      icon={jobsPage ? 'icon-history' : 'icon-undo'}
```

Comment:
I would always show the same icon, for consistency:

```suggestion
      icon={'icon-undo'}
```